### DISCUSSION TITLE
How to sum values from array in mantine form

### DISCUSSION THREAD

**Discussion by van-vince** [mantinedev/mantine#3738] [Q&A]
Please I want help to sum the price and quantity and also to calculate all the values in the sum column 
![Screenshot 2023-03-10 at 4 04 13 AM](https://user-images.githubusercontent.com/67305320/224220581-909a0bce-a3b5-40db-b485-48a718214fdf.png)
![Screenshot 2023-03-10 at 4 01 14 AM](https://user-images.githubusercontent.com/67305320/224220648-ecc02712-b855-4d92-8953-2d1e9e76fbcd.png)
![Screenshot 2023-03-10 at 4 00 07 AM](https://user-images.githubusercontent.com/67305320/224220669-710a1dd5-2c7c-4f21-99a1-0b688587c8f4.png)

**Comment by rtivital** [comment]
`const sum = form.values.invoiceDetails.reduce((acc, item) => acc + item.price, 0)`;

**Reply by van-vince** [reply]
Not working please.. I mean (sum = quantity * price ) for each row

**Reply by rtivital** [reply]
You can perform any operations on items inside reduce.
